.travis.yml

```diff
@@ -22,6 +22,6 @@ branches:
 
 script:
 - bundle exec jekyll build --config _config.yml,_config_prod.yml
-- gem install html-proofer
-- htmlproofer ./_site --only-4xx --allow-hash-href --alt-ignore '/.*/'
+#- gem install html-proofer
+#- htmlproofer ./_site --only-4xx --allow-hash-href --alt-ignore '/.*/'
```

Comment:
It was disabled because it somehow prevented me from adding a new page. Here is the mail I got from Mirko Bonadei:

It seems that it checks the generated link and it points to webrtc.org but since it hasn't published yet it gets the 404 Not Found.
That is annoying, please comment lines 25 and 26 in the .travis.yml file so you can deploy.

I will look into this in a few days.